src/OctoshiftCLI.Tests/ado2gh.Commands/AddTeamToRepoCommandTests.cs

```diff
@@ -2,10 +2,10 @@
 using System.CommandLine;
 using System.Threading.Tasks;
 using Moq;
-using OctoshiftCLI.Commands;
+using OctoshiftCLI.ado2gh.Commands;
```

Comment:
we need to address the namespaces in a separate PR I guess to be compliant with C#'s naming guidelines.

Reply:
when you say C# naming guidelines, is there a way to automatically enforce them in static analysis?

Reply:
There could be, I need to look it up, but `Rider` warns you about those 😃